apps/ledger-live-desktop/src/renderer/screens/market/MarketCoinScreen/index.tsx

```diff
@@ -224,7 +233,7 @@ export default function MarketCoinScreen() {
 
       startStakeFlow();
     },
-    [currency?.ticker, startStakeFlow],
+    [currency?.ticker, startStakeFlow, swapDefaultTrack],
```

Comment:
There is a small bug here, we need to use here: "stakeDefaultTrack", not "swapDefaultTrack"

Reply:
Great spot.